app/assets/javascripts/views/taggings_show.js

```diff
@@ -0,0 +1,51 @@
+$(document).ready(function() {
+  "use strict";
+
+  $(".related-content-item-entry .title").show();
+  $(".related-content-item-entry .value").hide();
+  $(".add-sortable-input").show();
+  $(".sortable-inputs").sortable();
+
+  $(".ordered-related-items").on("click", ".select2-search-choice-close", function() {
+    $(this).parents("li").remove();
+    return false;
+  });
+
+  // Prevent default behaviour of submitting form on enter, when adding
+  // a new related link.
+  $("#related_item_new_base_path").on("keypress", function() {
+    if (event.keyCode === 13) {
+      $("#lookup_base_path_button").click();
+      return false;
+    }
+  });
+
+  $("#lookup_base_path_button").on("click", function() {
+    var basePathInput = $("#related_item_new_base_path");
```

Comment:
Could you change the name of the variables that are jQuery objects to have `$` as prefix please?
Seems to be a common practice https://github.com/alphagov/styleguides/blob/master/js.md#jquery